CHANGELOG.md

```diff
@@ -8,3 +8,5 @@ and this project adheres to [Semantic Versioning](http://semver.org/spec/v2.0.0.
 ### Added
 - Example use case of spec-generated Ruby client. 
 [cybearl/conjur-openapi-spec#12](https://github.com/cyberark/conjur-openapi-spec/issues/12)
+- Added the /accounts endpoint to the spec file
+  [cyberark/conjur-openapi-spec#73](https://github.com/cyberark/conjur-openapi-spec/pull/73)
```

Comment:
```suggestion
- Added the /authenticators endpoint to the spec file
  [cyberark/conjur-openapi-spec#58](https://github.com/cyberark/conjur-openapi-spec/issues/58)
```
please link to the issue instead of the PR